agent/fw_drupal_common.c

```diff
@@ -294,3 +294,22 @@ void nr_drupal_headers_add(zval* arg, bool is_drupal_7 TSRMLS_DC) {
 
   nr_php_zval_free(&newrelic_headers);
 }
+
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO \
+    && !defined OVERWRITE_ZEND_EXECUTE_DATA
+void invoke_all_push_hook_stacks(zval* hook_copy) {
```

Comment:
I think the name `nr_drupal_{invoke_all_hooks_}stack{s}_push()` would be better here for the following reasons:
1. The `nr_` prefix follows the convention of scoping the function to public, internal function. This convention is described in point 3. of [our style guide](https://github.com/newrelic/newrelic-php-agent/blob/main/STYLEGUIDE.md)
2. The `drupal_` sub-prefix follows the undocumented, yet silently followed (although not always), convention of scoping the function to the module related to Drupal instrumentation. This convention, although undocumented, helps when searching for symbols related to particular functionality/features of the agent
3. The `{invoke_all_hooks_}stack{s}` part describe the object on which this function operates. The `invoke_all_hooks` part is optional (hence in curly braces), but helpful to more precisely describe the object on which this function operates - drupal invoke_all hooks stack. The `s` (plural form of `stack`) is also optional but also better describe that this function operates on two stacks - the hooks stack and the states stack. To sum up, it'll be fine to have this portion as just `stack` in its simplest form, but other variations are also acceptable, `invoke_all_stacks` being the most verbose variation.
4. The `(noun/object)_verb` convention (where the noun/object in this case is `{invoke_all_}stack{s}` follows the convention for util_stack.h APIs and this API for `invoke_all_stacks` is an extension to/concrete use of generic stack.

Similar remarks apply to `invoke_all_clean_hook_stacks`.

Using this name would make reading the code in fw_drupal.c module easier.

Reply:
renamed in [2592e6f](https://github.com/newrelic/newrelic-php-agent/pull/701/commits/2592e6fd08814c13540b734d72eb1928d80e1078)